### ISSUE TITLE
austin_tui won't start even when austin is on path

### ISSUE THREAD

**Issue by cool-RR** [P403n1x87/austin-tui#6]
```
c:\Users\Administrator\Desktop>austin
Usage: austin [-aCefms?V] [-i n_us] [-o FILE] [-p PID] [-t n_ms] [-x n_sec]
            [--alt-format] [--children] [--exclude-empty] [--full]
            [--interval=n_us] [--memory] [--output=FILE] [--pid=PID]
            [--sleepless] [--timeout=n_ms] [--exposure=n_sec] [--help]
            [--usage] [--version] command [ARG...]

              _   _
 __ _ _  _ __| |_(_)_ _
/ _` | || (_-<  _| | ' \
\__,_|\_,_/__/\__|_|_||_|
≡ƒñô austin version: 2.0.0

c:\Users\Administrator\Desktop>
c:\Users\Administrator\Desktop>austin-tui.exe -p 8600
🏁 Starting the Austin TUI ...
❌ Austin failed to start. Please make sure that the Austin binary is
available from the PATH environment variable and that the command line
arguments that you have provided are correct.
Python version is unknown.
```

**Comment by P403n1x87** [comment]
I cannot reproduce this issue. Testing with both cmd and powershell I can attach to a running interpreter. That is, I start Python in a prompt, grab its PID from the task monitor and do

~~~
austin-tui -p <pid>
~~~

and the TUI comes up as expected.

Can I ask you to double-check that you are using the right PID.

Thanks.

**Comment by cool-RR** [comment]
Tangent: Maybe austin_tui should write the traceback to a log so troubleshooting would be easier?

**Comment by cool-RR** [comment]
Confirmed it happens with a good PID. Regardless, even if the PID was bad the error message should have been different.

**Comment by cool-RR** [comment]
I removed your except clause to make a better traceback. Also used better_exceptions to add more info: https://gist.github.com/cool-RR/0936dc47ce351d0996c48a1fd21f08e2

**Comment by cool-RR** [comment]
I think you've done too much catching of exceptions, to the point where I need to dig deep to find the actual exception that happened. I'm not sure, but it might be "'(36) Python version is unknown.'". Can you generate this error locally and ensure that it prints this exception when it happens instead of swallowing it?

**Comment by P403n1x87** [comment]
Thanks for your investigation! Error reporting is definitely something to improve. May I ask you what version of Python you're using and how it was installed?

**Comment by cool-RR** [comment]
3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit
(AMD64)]

Windows 7. Installed with the standard installer.

**Comment by P403n1x87** [comment]
Does austin itself work? I.e. if you do

~~~
austin -p <pid>
~~~

do you get samples out? If it's really a "'(36) Python version is unknown.'", then this means that the austin binary is failing to identify the version of Python that it's trying to attach to.

**Comment by cool-RR** [comment]
I get this:

```
$ austin.exe -p 44040
              _   _
 __ _ _  _ __| |_(_)_ _
/ _` | || (_-<  _| | ' \
\__,_|\_,_/__/\__|_|_||_|
≡ƒñô austin version: 2.0.0
≡ƒÉì Python version: 0.0.0
Python version is unknown.
```

The 44040 PID is of a Python of an IPython shell of Python 3.8.1 on Windows
7. I got the same results for other Python scripts running on my system.

So you could split this into a separate bug: "If the Python version is
unknown, show this error in austin_tui instead of saying the binary isn't
found".

And then, I'd break down the "Python version is unknown" error to show more
details.

**Comment by P403n1x87** [comment]
I have tested with the standard 3.8.1 for Windows and I still managed to get Austin to work as expected. I managed to attach to a running interpreter invoked with `py` from `cmd`. I can see two options going forward:

1. Ensure that the right PID is being passed to Austin. I know you said you double-checked but one thing to be aware of is that a process could just be a wrapper that spawns the actual Python interpreter, whose PID is then to be used. One thing to try is the use of the `-C`/`--children` switch to ask Austin to try sampling the children of the process. The parent process itself can then be anything, as long as it has at least a child process that is an actual Python interpreter.
2. Compile Austin with debug enabled as documented in the README and attach the generated log to the issue.

**Comment by rajnishkumarindexnine** [comment]
Hello, 

I am trying to use austin package for performance testing using Pytest. I have installed windows .msi installer.
I have set the path(user variables) in env variables, Still I am getting below error:
![image](https://user-images.githubusercontent.com/82587717/169817431-f7bc08af-a115-449b-b6c6-6bcacb4b815d.png)

![image](https://user-images.githubusercontent.com/82587717/169817305-6943fde7-1d8c-493f-99ae-46e829ee8df6.png)

Help would be highly appreciated.
Regards,
Rajnish
